sand/extensions/search/wikidata_search.py

```diff
@@ -66,9 +66,13 @@ def find_class_by_name(self, search_text: str) -> List[SearchResult]:
         """
         request_params = self.get_class_search_params(search_text)
         api_data = requests.get(self.wikidata_url, request_params)
-        search_results = api_data.json()['query']['search']
         payload_results = []
 
+        if "error" not in api_data.json():
+            search_results = api_data.json()['query']['search']
+        else:
+            return payload_results
```

Comment:
- For error, we should return error to the user. 
- For empty query, return empty result

Reply:
Wikidata Api treats as an error, should we consider it as an error too ?